pkg/service/driver.go

```diff
@@ -17,12 +17,12 @@ type OvirtCSIDriver struct {
 	*ControllerService
 	*NodeService
 	nodeId      string
-	ovirtClient *ovirt.Client
+	ovirtClient ovirtclient.ClientWithLegacySupport
 	Client      client.Client
 }
 
 // NewOvirtCSIDriver creates a driver instance
-func NewOvirtCSIDriver(ovirtClient *ovirt.Client, client client.Client, nodeId string) *OvirtCSIDriver {
+func NewOvirtCSIDriver(ovirtClient ovirtclient.ClientWithLegacySupport, client client.Client, nodeId string) *OvirtCSIDriver {
```

Comment:
We should return the `Client` interface if possible, let's not give us / other people an out to fall back to the old SDK. This interface will also make it impossible to add mocks as the mock client only implements `Client`.